### PR TITLE
Fix webcam upload file type bug

### PR DESCRIPTION
**Pull request by hjheath**
The logic checking for File objects failed on Blobs.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://191-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-191-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/utils/onfidoApi.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ const formatError = ({response, status}, onError) =>
   onError({status, response: JSON.parse(response)})
 
 export const uploadDocument = (data, token, onSuccess, onError) => {
-  data['sdk_validations'] = {'detect_document': 'error', 'detect_glare': 'warn'}
+  const validations = {'detect_document': 'error', 'detect_glare': 'warn'}
+  data['sdk_validations'] = JSON.stringify(validations)
```
Comment (stefaniacardenas) [inline src/components/utils/onfidoApi.js]: Ricardo's comment on my PR: "better never to change objects in place unless it's made clear. a simple data= {...data, sdk_validations: ... will do the trick"

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-191-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://191-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**